humanguide/layout/03-main-actions.rst

```diff
@@ -0,0 +1,78 @@
+Main actions
+============
+
+Main actions are displayed in the corresponding header of the Page.
+Trying to be focused on the action relevant to the current page.
```

Comment:
"Trying to be focused on the action..." is a sentence fragment. Should it be "They should be focused..." instead?

Reply:
I'll rephrase this. What about «The header should show only relevant actions to the current page.»?